extensions/wikia/Wall/Wall.i18n.php

```diff
@@ -277,71 +277,6 @@
 
 	'wall-message-not-found-in-db' => 'We could not find this wall message in our database, sorry.',
 
-	/* Mail message */
-
-	'mail-notification-new-someone' => '$AUTHOR_NAME wrote a new message on $WIKI.',
```

Comment:
We have still have a bunch of these messages in this file. They're the non-english translations. Should we remove those as well?
```bash
$ ls Wall.i18n.php | ack --files-from=- -c mail-notification 
Wall.i18n.php:331
```